### PR TITLE
Changed NameNormaliser valid pattern.

### PR DESCRIPTION
**Pull request by nahguam**
# stream-registry PR

_&lt;High level description of the PR&gt;_

### Added
* _&lt;detail item of what was added&gt;_
* _&lt;describe functionality added&gt;_

### Changed
* _&lt;detail item of what was changed&gt;_
* _&lt;describe functionality that was changed&gt;
* _&lt;in particular, describe BACKWARD INCOMPATIBLE changes&gt;

### Deleted
* _&lt;detail item of what was removed&gt;_

# PR Checklist Forms

- [ ] CHANGELOG.md updated
- [ ] Reviewer assigned
- [ ] PR assigned (presumably to submitter)
- [ ] Labels added (enhancement, bug, documentation)

### CODE REVIEW

File: graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.expediagroup.streamplatform.streamregistry.graphql.model.inputs;
+
+import static com.expediagroup.streamplatform.streamregistry.graphql.model.inputs.NameNormaliser.normalise;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class NameNormaliserTest {
+  @Test
```
Comment (rshotels) [inline graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java]: I'm not sure if  abc_1_def is acceptable or should cause an exception. Would be nice to see a test which clarifies and validates the correct outcome.

File: graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  public void validName() {
```
Comment (rshotels) [inline graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java]: Seems like this filter would accept abc___________def (multiple underscores), which I suspect is not allowable. 
Would be nice to see a test which clarifies and validates this outcome.

File: graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java (hunk 1 above)
Comment (nahguam) [reply graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java]: Yes, should be acceptable - added a test

File: graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java (hunk 2 above)
Comment (nahguam) [reply graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java]: Good spot - updated regex to restrict consecutive underscores.

**Pull request merged**